src/uTensor/tensors/RamTensor.cpp

```diff
@@ -65,7 +65,19 @@ RamTensor::~RamTensor() {
 }
 
 void RamTensor::resize(TensorShape new_shape) {
```

Comment:
If resize is going to be supported in RamTensor we will need a test for it, 

Tensor A; A-> resize bigger;
Tensor A; Tensor B; A -> resize bigger; // Check for resize with large enough mem manager, B should be moved by the allocator to fit
Tensor A; Tensor B; A-> resize bigger; // Should throw OutOfMemError
Tensor A; A-> resize bigger; A-> resize to original size;

Reply:
Do you have example how to test for throwing errors?
Because current runtime does nothing when you throw an error via `Context`

Reply:
And how to test if allocator moves a `Handle`?

Reply:
https://github.com/uTensor/uTensor/blob/re-arch-argminmax-reshape/TESTS/tensors/test_ramtensor.cpp#L134-L185

The tests.